src/xenia/gpu/registers.h

```diff
@@ -289,12 +289,17 @@ union PA_SC_MPASS_PS_CNTL {
   static constexpr Register register_index = XE_GPU_REG_PA_SC_MPASS_PS_CNTL;
 };
 
-// Scanline converter viz query
+// Scanline converter viz query, used by D3D for gpu side conditional rendering
 union PA_SC_VIZ_QUERY {
   struct {
-    uint32_t viz_query_ena : 1;          // +0
-    uint32_t viz_query_id : 6;           // +1
-    uint32_t kill_pix_post_early_z : 1;  // +7
+    // +0 the visibility of draws should be evaluated
+    uint32_t viz_query_ena : 1;
+    // +1
```

Comment:
I meant just usage comments on separate lines, but offsets on the same lines as fields.